backoffice/workflows/api/views.py

```diff
@@ -70,6 +78,33 @@ def create(self, request, *args, **kwargs):
             return Response({"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+class AuthorWorkflowViewSet(viewsets.ViewSet):
+
+    serializer_class = WorkflowSerializer
+
+    def create(self, request):
+
+        serializer = self.serializer_class(data=request.data)
+
+        if serializer.is_valid(raise_exception=True):
+            workflow = Workflow.objects.create(
+                data=serializer.validated_data["data"], workflow_type=serializer.validated_data["workflow_type"]
+            )
+
+        return airflow_utils.trigger_airflow_dag(WORKFLOW_DAG[workflow.workflow_type], str(workflow.id), workflow.data)
```

Comment:
This line is exactly 119 which is the line length set for black:
https://github.com/inspirehep/backoffice/blob/main/pyproject.toml#L26 
Should this setting be removed from `pyproject.toml` so that the default 88 length is used?

Reply:
No it's ok, we can leave it as is